USAGE_desktop.md

```diff
@@ -546,6 +547,26 @@ Optional arguments:
   --version             Print version information and exit.
 ```
 
+### To ASCII
+
+The `gfxrecon-toascii.exe` converts GFXReconstruct capture files to text.
+The text output is formatted as JSON and written to stdout.
```

Comment:
On my machine with `dev` 000527572401079ca9b522490094dee9afda6121 (HEAD at the moment) the output is stored in a file with the suffix "gfxr" replaced with "txt".  Could you update this to indicate the output is written to a file?
```
$ ./build/tools/toascii/Debug/gfxrecon-toascii.exe ./gfxrecon_capture_20210809T163504.gfxr

$ ls -al gfxrecon*
-rw-r--r-- 1 grantham 197121 155120 Aug  9 16:35 gfxrecon_capture_20210809T163504.gfxr
-rw-r--r-- 1 grantham 197121 298127 Aug 12 08:54 gfxrecon_capture_20210809T163504.txt

$ 
```

Reply:
Done